.github/workflows/deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: Deploy to WordPress.org
+name: Deploy to WordPress.org and AWS S3
 on:
   release:
     types: [published]
```

Comment:
Before, there were two files, both of which would execute when you performed a release with a new draft for WordPress org. One file would create a zip and store it in AWS, and the second one would upload it to SVN and release the new version on org. Currently, as per our discussion, we are merging these two files and performing the same action when you draft a new version and release it on org.

Reply:
I know that we're changing the workflow since this is the change on the workflow that I wrote down. I also understand the difference between the types `published` and `released`. I'm asking why it's treated differently here than in the other file. I want to ensure I fully understand the rationale behind the specific uses in different contexts.